pages/general/adopting.md

```diff
@@ -0,0 +1,11 @@
+# Adopting AEPs for your company
+
+Currently, we encourage adoption of AEPs via having your APIs adhere to the
+guidance served at https://aep.dev.
+
+In the future, we plan to introduce a broader ecosystem of tooling including:
+
+- A forkable repository which allows for internal mirrors and
```

Comment:
Doesn't this conflict somewhat with the FAQ explanation that forks are discouraged?

Reply:
We already have a forkable repository (every repo on GH is forkable) - so what exactly would we be introducing here?

Reply:
I'll remove for now so we can get the rest of this merged in.

I was trying to say we'll have a model where you can have extensible AEPs without violating the core ones - but I can work on that wording later.